pkg/rke2/clusterrole_templates.go

```diff
@@ -0,0 +1,70 @@
+package rke2
+
+const (
+	kubeletAPIServerRoleBindingName = "kube-apiserver-kubelet-admin"
+	tunnelControllerRoleName        = "system:k3s-controller"
```

Comment:
thought this was going to be changing?

Reply:
Yes, I noted that in the PR description. If the k3s PR gets accepted and pulled through to rke2 then I'll update it before merging, otherwise it will need to be changed after that's done.

Reply:
I've added a k3s update to this PR to address this.